src/bayesdb.py

```diff
@@ -377,6 +377,19 @@ def changes(self):
         """
         return self._sqlite3.changes()
 
+    def dump_models(self, population_name, path):
+        """Write information about a population's models to a file, as JSON.
+
+        The intent is that this information, combined with the data
+        table (not itself dumped here), should be sufficient to simulate 
+        any of the models.
+        """
+        import json
```

Comment:
Imports should be at top level.

Reply:
Why?

Reply:
PEP 8 "Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants."  Obviously there are exceptions to this rule in some circumstances, but `import json` is not one of them.